src/services/GrantsService.ts

```diff
@@ -160,4 +174,34 @@ export class GrantsService {
 
     return { ...currentUpdate, index: publicUpdates.length }
   }
+
+  public static async getGrantInCreation(grantRequest: GrantRequest, user: string) {
+    const grantSize = Number(grantRequest.funding || 0)
+
+    await BudgetService.validateGrantRequest(grantSize, grantRequest.category)
+
+    const grantConfiguration: GrantProposalConfiguration = {
+      ...grantRequest,
+      size: Number(grantRequest.funding),
+      tier: GrantTier.getTypeFromBudget(grantSize),
+      choices: DEFAULT_CHOICES,
+    }
+
+    const grantInCreation: ProposalInCreation = {
+      user,
+      type: ProposalType.Grant,
+      required_to_pass: GrantTier.getVPThreshold(grantSize),
+      finish_at: this.getFinishAt(),
+      configuration: grantConfiguration,
+    }
+    return grantInCreation
+  }
+
+  private static getFinishAt() {
+    const GATSBY_GRANT_PROPOSAL_DURATION_IN_SECONDS = env('GATSBY_GRANT_PROPOSAL_DURATION_IN_SECONDS')
+    if (isDevEnv() && GATSBY_GRANT_PROPOSAL_DURATION_IN_SECONDS) {
```

Comment:
Is this necessary? We are using `GATSBY_GRANT_PROPOSAL_DURATION_IN_SECONDS` in all cases 🤔